src/app/docs/button/button-docs.component.html

```diff
@@ -0,0 +1,204 @@
+              <button 
+                type="button" 
+                class="sky-btn sky-btn-default"
+              >
+                Secondary
+              </button>
+            </td>
+            <td class="sky-docs-table-cell">
+              <p>
+                Secondary buttons trigger less important or less frequently used actions. SKY UX reduces the visual weight to make them less attention-drawing than the primary buttons.
+              </p>
+            </td>
+            <td class="sky-docs-table-cell">
+              <p>
+                <sky-code>class=&quot;sky-btn sky-btn-default&quot;</sky-code>
+              </p>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              <button 
+                type="button" 
+                class="sky-btn sky-btn-link"
+              >
+                Link
+              </button>
+            </td>
+            <td class="sky-docs-table-cell">
+              <p>
+                Link buttons trigger actions that take users to other pages or actions that end tasks or exit modals (such as <strong>Cancel</strong>).
+              </p>
+            </td>
+            <td class="sky-docs-table-cell">
+              <p>
+                <sky-code>class=&quot;sky-btn sky-btn-link&quot;</sky-code>
+              </p>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              <button 
+                type="button" 
+                class="sky-btn sky-btn-link-inline"
+              >
+                Inline link
+              </button>
+            </td>
+            <td class="sky-docs-table-cell">
+              <p>
+                Inline link buttons trigger the same type of actions as link buttons, but the styling is altered to fit inline.
+              </p>
+            </td>
+            <td class="sky-docs-table-cell">
+              <p>
+                <sky-code>class=&quot;sky-btn sky-btn-link-inline&quot;</sky-code>
+              </p>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              <div style="margin-bottom: 10px;">
+                <button 
+                  type="button" 
+                  class="sky-btn sky-btn-primary sky-btn-disabled"
+                >
+                  Disabled
+                </button>
+              </div>
+              <div style="margin-bottom: 10px;">
+                <button 
+                  type="button" 
+                  class="sky-btn sky-btn-default sky-btn-disabled"
+                >
+                  Disabled
+                </button>
+              </div>
+              <div style="margin-bottom: 10px;">
+                <button 
+                  type="button" 
+                  class="sky-btn sky-btn-link sky-btn-disabled"
+                >
+                  Disabled
+                </button>
+              </div>
+            </td>
```

Comment:
Looks like we lost a disabled inline link button. Was that intentional?

Reply:
Nope, that was unintentional. Added back :)